molgenis-swagger/src/main/resources/templates/view-swagger.ftl

```diff
@@ -0,0 +1,194 @@
+<#-- @formatter:off -->
+swagger: '2.0'
+info:
+  title: MOLGENIS REST API
+  description: Documentation for the MOLGENIS Rest API V2
+  version: "v2"
+host: molgenis01.gcc.rug.nl
```

Comment:
Do not hardcore molgenis01.gcc.rug.nl as host, the swagger-ui should be based on the host it is running on.